SYCL/ESIMD/api/functional/operators/operator_decrement_and_increment_fp_extra.cpp

```diff
@@ -0,0 +1,51 @@
+//==------- operator_decrement_and_increment_fp_extra.cpp  - DPC++ ESIMD
+//          on-device test -------------------------------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu, level_zero
+// XREQUIRES: gpu
+// TODO gpu and level_zero in REQUIRES due to only this platforms supported yet.
+// The current "REQUIRES" should be replaced with "gpu" only as mentioned in
+// "XREQUIRES".
+// UNSUPPORTED: cuda, hip
+// RUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+//
+// TODO simd<sycl::half, N> vector filled with unexpected value in case current
+// value is equal to -nan.
+//
+// Test for simd increment and decrement operators.
+// The test creates source simd instance and call increment or decrement
+// operator opera with reference data. The test verifies that in the output data
+// contained correctness data according to chosen operator.
+
+#include "operator_decrement_and_increment.hpp"
+
+using namespace esimd_test::api::functional;
+
+int main(int, char **) {
+  sycl::queue queue(esimd_test::ESIMDSelector{},
+                    esimd_test::createExceptionHandler());
+
+  bool passed = true;
+#ifdef RUN_TEST_WITH_HALF
```

Comment:
```suggestion
#ifdef ESIMD_TESTS_RUN_WITH_HALF
```

Reply:
this comment was applied [here](https://github.com/intel/llvm-test-suite/pull/827/commits/07503714b6bf2d29f8318597e51142bec03a1fb8#diff-8bfd87b59402abfe7e7ca599205da194d65cb2a3f3dbf151a6687b21baf62758L35-L36)